src/.htaccess

```diff
@@ -1 +1,4 @@
 RewriteEngine On
+
+RewriteCond %{HTTPS} !=on
+RewriteRule ^/?(.*) https://%{SERVER_NAME}/$1 [R,L]
```

Comment:
The `SERVER_NAME` Server-Variable maps to the [ServerName Directive|https://httpd.apache.org/docs/current/mod/core.html#servername]. The syntax for this directive says that it can optionally include the scheme. This may cause problems with the rewrite rule if a scheme is included in the server name. Is it worth using the `HTTP_HOST` instead? The downside I see to this is that `HTTP_HOST` is the host name only where as the `SERVER_NAME` can include the port number.

Reply:
have tested it. `SERVER_NAME` works fine in different scenarios.